editors/blocknote-headless/src/blocknote/uniast-to-bn.ts

```diff
@@ -285,9 +285,7 @@ export class UniAstToBlockNoteConverter {
         url,
         caption: image.caption ?? "",
         showPreview: true,
-        // NOTE: 512 is the default width applied by BlockNote when inserting images in the editor
-        //       or when converting from Markdown / HTML
-        previewWidth: image.widthPx ?? 512,
+        previewWidth: image.widthPx ?? (undefined as unknown as number),
```

Comment:
Why not just this? `widthPx` is already optional and can be `undefined`
```suggestion
        previewWidth: image.widthPx
```

Reply:
Checking in my IDE, the compiler is indeed not happy with my proposed change. It looks like a bug in Blocknote types declaration.

Reply:
Yes, as indicated in our `cristal` channel, there is a typing issue in BlockNote
Relevant issue I opened 2 days ago: https://github.com/TypeCellOS/BlockNote/issues/1765